edgedb/protocol/protocol.pyx

```diff
@@ -807,7 +807,8 @@ cdef class SansIOProtocol:
                 self.parse_headers()
                 self.buffer.finish_message()
 
-                if major != PROTO_VER_MAJOR or minor < PROTO_VER_MINOR_MIN:
+                if (major != PROTO_VER_MAJOR or
+                        (major == 0 and minor != PROTO_VER_MINOR)):
```

Comment:
`major == 0` is always `True`, because `PROTO_VER_MAJOR` is `0`? I'm asking because I'm bumping `PROTO_VER_MAJOR` to `1`, and I don't want to miss a check here.

Reply:
The idea is that when "major" is zero minor version is treated as major.

So, yes, drop the second check when you bump `PROTO_VER_MAJOR` to >= 1.